django_project/feti/static/feti/css/landing-page.css

```diff
@@ -546,7 +546,7 @@
 .result-title img {
     width: 54px;
     height: auto;
-    margin-right: 20px;
+    margin: 10px 5px 5px 5px;
```

Comment:
I see. 
Yeah I've been looking how to do it, since even though I added `vertical-align: middle` in css file, it didn't center the icon. Most probably because the icon use `float: left`
I'll look into it again.